doc/source/conf.py

```diff
@@ -160,15 +160,13 @@ def intersphinx_pyansys_geometry(switcher_version: str):
             "icon": "fa fa-file-pdf fa-fw",
         },
     ],
-    "use_meilisearch": {
-        "api_key": os.getenv("MEILISEARCH_PUBLIC_API_KEY", ""),
-        "index_uids": {
-            f"pyansys-geometry-v{get_version_match(__version__).replace('.', '-')}": "PyAnsys-Geometry",  # noqa: E501
-        },
-    },
     "ansys_sphinx_theme_autoapi": {
         "project": project,
     },
+    "static_search": {
+        "threshold": 0.5,
+        "min_chars_for_search": 2,
```

Comment:
```suggestion
        "min_chars_for_search": 2,
        "ignoreLocation": True,
```
For smaller libraries with fewer documents, we can use this option without impacting search time while achieving better results. Feel free to disregard, up to you: [Fuse.js - `ignoreLocation` option](https://www.fusejs.io/api/options.html#ignorelocation).

Reply:
Cool, didn't know that! Thanks @Revathyvenugopal162!

Reply:
we can use any options from Fuse.js in our theme options depending on the size of docs.